src/Mod/Tux/NavigationIndicatorGui.py

```diff
@@ -799,33 +799,33 @@ def onTooltip():
 def onOrbit():
     """Use turntable or trackball orbit style."""
 
-    if aTurntable.isChecked():
+    if aRoundedArcball.isChecked():
         pView.SetInt("OrbitStyle", 0)
     elif aTrackball.isChecked():
         pView.SetInt("OrbitStyle", 1)
-    elif aFreeTurntable.isChecked():
-        pView.SetInt("OrbitStyle", 2)
     elif aTrackballClassic.isChecked():
+        pView.SetInt("OrbitStyle", 2)
+    elif aTurntable.isChecked():
         pView.SetInt("OrbitStyle", 3)
-    elif aRoundedArcball.isChecked():
+    elif aFreeTurntable.isChecked():
         pView.SetInt("OrbitStyle", 4)
```

Comment:
Instead of this, create an enum which mirror the cpp enum (we currently have no better option at the moment) and use the enum values instead of these hardcoded ones.

We don't need to change the order of the ifs either. It is good to keep them in the enum order